### PR TITLE
Student validation fix & schedule refactoring& new endpoint & schedule docs completed

### PR DESCRIPTION
**Pull request by Crutovel**

### CODE REVIEW

File: doc/students.md [hunk 1]
```diff
@@ -366,7 +396,7 @@ application/json
     }
 
     {
-        "message": "Access Denied 
+        "message": "Access Denied" 
     }
 
 [Table of content](#table-of-content)
```
Comment (khdimon) [inline doc/students.md]: Need to change "add" on "edit" in 405 line

File: doc/students.md [hunk 2]
```diff
@@ -392,15 +422,31 @@ None
```
Comment (khdimon) [inline doc/students.md]: We have 2 endpoints: /students (PUT) for edit list of student and /students/{id} (PUT) for edit one student. I think it need to remove "{id}" in line 409 and add separately info about endpoint /students/{id} (PUT)

Comment (Crutovel) [inline doc/students.md]: Sure thx!

File: doc/students.md (hunk 1 above)
Comment (Crutovel) [inline doc/students.md]: Yeah! thanks

File: doc/schedule.md [hunk 3]
```diff
@@ -797,5 +813,259 @@ application/json
...
+<a name="lastWeekEvents"></a>
+## Show lastWeekEvents
+
+<a name="lastWeekEvents-url"></a>
+### URL :
+/events/copypaste
+
+<a name="lastWeekEvents-method"></a>
+### Method :
+POST
```
Comment (thedikiy) [inline doc/schedule.md]: GET

File: src/main/resources/i18n/messages.properties [hunk 4]
```diff
@@ -36,6 +37,10 @@ illegalArgs.keyDate.event.group=Group is not specified
 illegalArgs.student.cvOrImage.name=Sorry, you should use only allowed file types: doc, docx, pdf or rtf for CV and jpg, jpeg, tiff or png for photo
 illegalArgs.student.image.presentName=You must send image name with image
 illegalArgs.student.cv.presentName=You must send CV name with CV
+illegalArgs.student.firstName=first name must be specified
+illegalArgs.student.lastName=last name must be specified
+illegalArgs.student.englishLevel="English level must be specified"
+illegalArgs.student.expert="Expert must be specified"
```
Comment (thedikiy) [inline src/main/resources/i18n/messages.properties]: Quotes are not necessary

File: doc/students.md [hunk 5]
```diff
@@ -392,15 +562,31 @@ None
```
Comment (khdimon) [inline doc/students.md]: Please add link to the table of content at the end of file

Comment (Crutovel) [inline doc/students.md]: ok!

**Pull request merged**